go.mod

```diff
@@ -28,6 +27,7 @@ require (
 	github.com/go-asn1-ber/asn1-ber v1.5.1 // indirect
 	github.com/golang/protobuf v1.4.2 // indirect
 	github.com/golang/snappy v0.0.1 // indirect
+	github.com/hashicorp/errwrap v1.1.0 // indirect
```

Comment:
Looks like we removed the sole spot this package was used. Is it safe to no longer include this package in `go.mod` or are there build/consistency reasons for which we should keep them around?

Reply:
This is an indirect/transitive dependency. `go mod tidy` doesn't yield a diff. 

It comes from go-multierror:
```sh
go mod why github.com/hashicorp/errwrap

# github.com/hashicorp/errwrap
github.com/hashicorp/vault-plugin-secrets-openldap
github.com/hashicorp/go-multierror
github.com/hashicorp/errwrap
```

Reply:
Ah gotcha, thanks for confirming!